ArcadiaParties-SPA/src/app/user/user-info/user-info.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="flex-container">
+  <mat-card>
+    <mat-card-title class="center">User Profile</mat-card-title>
+    <mat-card-content *ngIf="user$ | async as user; else emptyUser">
+      <mat-tab-group mat-stretch-tabs>
+        <mat-tab label="General">
+          <div class="center">
+            <p>Identity: {{ user.identity }} </p>
+            <p>FirstName: {{ user.firstName }} </p>
+            <p>LastName: {{ user.lastName }} </p>
+            <p>Department: {{ user.department }} </p>
+            <p>Roles: <span *ngFor="let role of user.roles"> {{ role }} </span></p>
+          </div>
+        </mat-tab>
+        <mat-tab label="Dates">
+          <p class="center" *ngFor="let date of user.dates">{{ date.name }}:
+            {{ date.date | date:'longDate' }} </p>
+        </mat-tab>
+      </mat-tab-group>
+      <!-- <button mat-raised-button color="primary" class="edit-button">Edit</button> -->
```

Comment:
Remove it. When you need it - then you can add it again.

Reply:
It removed